### DISCUSSION TITLE
blog/2024/09/20/%E6%91%B8%E9%B1%BC%E5%BF%8D%E8%80%85-%E9%BB%98%E9%BB%98%E6%93%8D%E4%BD%9C/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#20] [General]
# blog/2024/09/20/%E6%91%B8%E9%B1%BC%E5%BF%8D%E8%80%85-%E9%BB%98%E9%BB%98%E6%93%8D%E4%BD%9C/

用suno试了下chatGPT的歌词，太震惊我了，我去！！！！

https://996.ninja/blog/2024/09/20/%E6%91%B8%E9%B1%BC%E5%BF%8D%E8%80%85-%E9%BB%98%E9%BB%98%E6%93%8D%E4%BD%9C/

**Comment by 67haha** [comment]
不是？还蛮好听的啊

**Reply by MingGH** [reply]
我也觉着，甚至还听了好几遍。。
